lib/routes/autocentre/index.ts

```diff
@@ -0,0 +1,32 @@
+import { Data, Route } from '@/types';
+import parser from '@/utils/rss-parser';
+import cache from '@/utils/cache';
+
+export const route: Route = {
+    path: '',
+    name: 'Автоцентр.ua',
```

Comment:
name should not be the same as namespace's name

Reply:
Could you please suggest what I should put here? I haven't noticed it in the Documentation.

Reply:
Done